utils/docker-compose/network-config/base/base.yaml

```diff
@@ -37,6 +38,7 @@ services:
         - /var/run/:/host/var/run/
         - ../config:/etc/hyperledger/configtx
         - ../../../../examples:/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples
```

Comment:
I like that idea, but I have to play around a little before I know what would be the best way to approach it. The different paths was one of the reasons I mounted the entire demo directory instead of just the golang chaincode. 

The other issue is with different paths, if they have to be mounted to different locations, then the rest of the scripts/instructions might not work.